lxd/container.go

```diff
@@ -456,7 +450,7 @@ func containerLXDLoad(d *Daemon, name string) (container, error) {
 		baseConfig:   baseConfig,
 		baseDevices:  baseDevices}
 
-	s, err := storageForFilename(d, c.PathGet(""))
+	s, err := storageForFilename(d, shared.VarPath("containers", strings.Split(name, "/")[0]))
```

Comment:
This seems like it could use a comment. The commit message is clear, but just reading through the source, it might be confusing. I won't hold up the branch for that though.